### PR TITLE
Np 45128 aggregation for institution

### PR DESCRIPTION
**Pull request by truhacevkir**

### CODE REVIEW

File: index-handlers/src/main/java/no/sikt/nva/nvi/index/Aggregations.java [hunk 1]
```diff
@@ -1,62 +1,95 @@
...
-    private static final String APPROVED = "approved";
-    private static final String REJECTED = "rejected";
     private static final CharSequence JSON_PATH_DELIMITER = ".";
-    public static final Map<String, Aggregation> AGGREGATIONS_MAP = Map.of(
-        jsonPathOf(APPROVAL_STATUS, PENDING), pendingAggregation(),
-        jsonPathOf(APPROVAL_STATUS, APPROVED), approvedAggregation(),
-        jsonPathOf(APPROVAL_STATUS, REJECTED), rejectedAggregation()
-    );
+    public static final String FOR_CONTROL_AGGREGATION_NAME = "for_control";
+    public static final String FOR_CONTROL_MULTIPLE_APPROVALS_AGGREGATION_NAME = "for_control_multiple_approvals";
```
Comment (truhacevkir) [inline index-handlers/src/main/java/no/sikt/nva/nvi/index/Aggregations.java:26]: unsure about naming of these two aggregations

File: index-handlers/src/test/resources/document_rejected.json [hunk 2]
```diff
@@ -1,6 +1,9 @@
 {
   "@context": "https://bibsysdev.github.io/src/nvi-context.json",
-  "identifier": "01892ad6fc07-1234234-657-78698677",
+  "identifier": "01892ad6fc07-3145-4bf77-d4202d074380",
+  "year": "2023",
```
Comment (monaullah) [inline index-handlers/src/test/resources/document_rejected.json:4]: Did we not remove "year"?

File: index-handlers/src/main/java/no/sikt/nva/nvi/index/Aggregations.java [hunk 3]
```diff
@@ -1,62 +1,137 @@
...
+    private static Aggregation forControlAssignedMultipleApprovalsAggregation(String customer) {
+        var customerQuery = termQuery(customer, jsonPathOf(APPROVALS, ID));
+        var approvalStatusQuery = termQuery(PENDING.getValue(), jsonPathOf(APPROVALS, APPROVAL_STATUS));
+        var assigneeQuery = existsQuery(jsonPathOf(APPROVALS, ASSIGNEE));
+        var multipleAffiliationsQuery = mustMatch(rangeFromQuery(NUMBER_OF_APPROVALS, 2));
+
+        var nestedQuery = new NestedQuery.Builder()
+                              .path(APPROVALS)
+                              .query(mustMatch(assigneeQuery, customerQuery, approvalStatusQuery))
+                              .build()._toQuery();
```
Comment (brinxmat) [inline index-handlers/src/main/java/no/sikt/nva/nvi/index/Aggregations.java:52]: is there is a pattern being missed?

**Review by brinxmat — CHANGES_REQUESTED** [review]

File: index-handlers/src/main/java/no/sikt/nva/nvi/index/Aggregations.java (hunk 3 above)
Comment (truhacevkir) [reply index-handlers/src/main/java/no/sikt/nva/nvi/index/Aggregations.java]: Have refactored a bit

**Review by sondrev — APPROVED** [review]

**Pull request merged**